wagtail/utils/deprecation.py

```diff
@@ -2,5 +2,8 @@ class RemovedInWagtail12Warning(DeprecationWarning):
     pass
 
 
+removed_in_next_version_warning = RemovedInWagtail12Warning
```

Comment:
Any reason why we need this alias, rather than just setting the filter on DeprecationWarning? Are there other types of DeprecationWarning that we don't want to show?